docs/releases/2.2.txt

```diff
@@ -171,7 +171,8 @@ Generic Views
 Internationalization
 ~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* Changed :setting:`LOCALE_PATHS` setting to allow usage of custom gettext
+  domains (:ticket:`6376`).
```

Comment:
Move to 3.0.txt

Reply:
Also omit the ticket reference, that's reserved for bug fixes. "The LOCALE_PATHS setting now supports custom gettext domains." (perhaps with a link to an explanation of what functionality custom gettext domains offers -- maybe that's beyond the scope of Django's documentation but as someone who doesn't use translations, the purpose of custom domains doesn't seem to be described anywhere in these changes.)